### ISSUE TITLE
Can't install on OS X

### ISSUE THREAD

**Issue by ronilaukkarinen** [melchor629/node-chromecaster-cli#1]
Output:

````
10 warnings and 8 errors generated.
make: *** [Release/obj.target/AudioInputNative/src/wrappers.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Darwin 15.4.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/chromecaster-cli/node_modules/chromecaster-lib
gyp ERR! node -v v0.12.7
gyp ERR! node-gyp -v v2.0.1
gyp ERR! not ok 
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "chromecaster-cli"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3
npm ERR! code ELIFECYCLE

npm ERR! chromecaster-lib@0.3.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the chromecaster-lib@0.3.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the chromecaster-lib package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls chromecaster-lib
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/rolle/npm-debug.log
````

Using El Capitan, 10.11.4.

Last lines of long npm-debug.log: 

````
6152 error Exit status 1
6153 error Failed at the chromecaster-lib@0.3.0 install script 'node-gyp rebuild'.
6153 error This is most likely a problem with the chromecaster-lib package,
6153 error not with npm itself.
6153 error Tell the author that this fails on your system:
6153 error     node-gyp rebuild
6153 error You can get their info via:
6153 error     npm owner ls chromecaster-lib
6153 error There is likely additional logging output above.
6154 verbose exit [ 1, true ]
6155 verbose unbuild lib/node_modules/chromecaster-cli/node_modules/chromecaster-lib
6156 info preuninstall chromecaster-lib@0.3.0
6157 info uninstall chromecaster-lib@0.3.0
6158 verbose unbuild rmStuff chromecaster-lib@0.3.0 from /usr/local/lib/node_modules
6159 verbose unbuild rmStuff in /usr/local/lib/node_modules/chromecaster-cli/node_modules
6160 info postuninstall chromecaster-lib@0.3.0
6161 silly gentlyRm /usr/local/lib/node_modules/chromecaster-cli/node_modules/chromecaster-lib is being purged from base /usr/local
6162 verbose gentlyRm don't care about contents; nuking /usr/local/lib/node_modules/chromecaster-cli/node_modules/chromecaster-lib
6163 silly vacuum-fs purging /usr/local/lib/node_modules/chromecaster-cli/node_modules/chromecaster-lib
6164 silly vacuum-fs quitting because other entries in /usr/local/lib/node_modules/chromecaster-cli/node_modules
6165 verbose unbuild lib/node_modules/chromecaster-cli
6166 info preuninstall chromecaster-cli@0.2.1
6167 info uninstall chromecaster-cli@0.2.1
6168 verbose unbuild rmStuff chromecaster-cli@0.2.1 from /usr/local/lib/node_modules
6169 info postuninstall chromecaster-cli@0.2.1
6170 silly gentlyRm /usr/local/lib/node_modules/chromecaster-cli is being purged from base /usr/local
6171 verbose gentlyRm don't care about contents; nuking /usr/local/lib/node_modules/chromecaster-cli
6172 silly vacuum-fs purging /usr/local/lib/node_modules/chromecaster-cli
6173 silly vacuum-fs quitting because other entries in /usr/local/lib/node_modules
````

**Comment by melchor629** [comment]
Thanks for the report! 😄 I just post a fix for the compilation (tested on node 0.12.14).

**Comment by ronilaukkarinen** [comment]
Thanks for the fix, it works now!
